### PR TITLE
Feature/168 random stories

### PR DESCRIPTION
**Pull request by vinovo**
Added an endpoint for fetching k user stories at country/state/city scope

### CODE REVIEW

File: frontend/src/utils.js [hunk 1]
```diff
@@ -71,4 +71,26 @@ export const snakeToCamelCase = (str) =>
...
+      properties: properties,
+    });
+  });
+
+  return {
+    type: "FeatureCollection",
+    features: features,
+  };
+};
+
```
Comment (vinovo) [inline frontend/src/utils.js:95]: @yus252  the code for converting story array to geojson object is here. you might need to modify it a little bit to fit your need...

Comment (yus252) [reply frontend/src/utils.js]: I c. Thanks!

File: backend/alembic/versions/2551e1a3cb06_add_spam_column_to_stories.py [hunk 2]
```diff
@@ -20,7 +20,21 @@ def upgrade():
     op.add_column(
         "stories", sa.Column("spam", sa.Integer(), server_default="0")
     )
+    op.add_column(
```
Comment (yus252) [inline backend/alembic/versions/2551e1a3cb06_add_spam_column_to_stories.py:23]: @vinovo  Can you verify it?

Comment (vinovo) [reply backend/alembic/versions/2551e1a3cb06_add_spam_column_to_stories.py:23]: lgtm. i just removed the migration version w/ conflict, @yus252 @TanayKarve can you try the upgrade head command on your end?

File: frontend/src/utils.js [hunk 3]
```diff
@@ -71,4 +71,27 @@ export const snakeToCamelCase = (str) =>
...
 const camelToSnakeCase = (str) => str.replace(/([A-Z])/g, "_$1").toLowerCase();
 
+export const storiesToGeoJson = (stories) => {
+  stories = stories.filter(
+    (story) => story && story.latitude && story.longitude
+  );
+
+  let features = stories.map((story) => {
+    let { latitude, longitude, ...properties } = story;
+    return {
```
Comment (vinovo) [inline frontend/src/utils.js:81]: we could add a line for generating random number for lat&lon here, i.e. `latitude += random(); longitude += random();`

Comment (yus252) [reply frontend/src/utils.js]: Added

**Comment by yus252** [comment]
@ste519 Need to change the background map into picture.

**Comment by yus252** [comment]
We don't need random story feature anymore and rendering all stories's code is in feature/161-add-story-marker.

**Pull request closed without merging**